templates/_components/_image.twig

```diff
@@ -500,9 +500,9 @@
     'data-src': imageDefault.url,
 
   }:{}) | merge(imageTransforms is defined and imageTransforms|length > 1 ? {
-    sizes: imageSizesGuide ?? false,
+    sizes: imageLazyload ? 'auto' : (imageSizesGuide ?? false),
```

Comment:
Why are we setting this to `auto` for lazyloaded images? `auto` is not a valid value for the `sizes` attribute, it is only used by lazysizes via the `data-sizes` attribute.

Reply:
Sorry, mixed the values of the `sizes` and `data-sizes` attributes. Fixed now.